evap/staff/fixtures/minimal_test_data.json

```diff
@@ -1181,6 +1181,58 @@
   "pk": 36,
   "model": "evaluation.contribution",
   "fields": {
+    "contributor": 2,
```

Comment:
they were inconsistent before. multiple courses had no responsible contribution and/or general contribution.